### PR TITLE
ci: Fix devnet

### PR DESCRIPTION
**Pull request by mslipper**
Various fixes to make the devnet run again.

### CODE REVIEW

**Comment by mslipper** [comment]
Missed a comma.

**Review by trianglesphere — APPROVED** [review]
lgtm. FYI default auth port is 8551

File: ops-bedrock/docker-compose.yml [hunk 1]
```diff
@@ -46,7 +47,7 @@ services:
     command: >
       op-node
       --l1=ws://l1:8546
-      --l2=ws://l2:8546
+      --l2=http://l2:8547
```
Comment (protolambda) [inline ops-bedrock/docker-compose.yml:50]:
```suggestion
      --l2=http://l2:8551
```

File: ops-bedrock/entrypoint.sh [hunk 2]
```diff
@@ -60,5 +60,9 @@ exec geth \
 	--miner.etherbase=$BLOCK_SIGNER_ADDRESS \
 	--password="$GETH_DATA_DIR"/password \
 	--allow-insecure-unlock \
+	--authrpc.addr="0.0.0.0" \
+	--authrpc.port="8547" \
```
Comment (protolambda) [inline ops-bedrock/entrypoint.sh:64]:
```suggestion
	--authrpc.port="8551" \
```

**Review by protolambda — APPROVED** [review]
LGTM. Maybe use the default engine API auth port instead

**Comment by mslipper** [comment]
Done, committed.

**Review by tynes — APPROVED** [review]

**Pull request merged**